packages/datadog_inappwebview_tracking/lib/src/datadog_inappwebview_user_script.dart

```diff
@@ -0,0 +1,93 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-Present Datadog, Inc.
+
+import 'package:datadog_flutter_plugin/datadog_flutter_plugin.dart';
+import 'package:datadog_flutter_plugin/datadog_internal.dart';
+import 'package:flutter_inappwebview/flutter_inappwebview.dart';
+
+import 'datadog_inappwebview_tracking_platform_interface.dart';
+
+const handlerName = 'datadog_inappwebivew_handler';
+
+String _createBridgeSource(
+    DatadogSdk datadog, String bridgeName, Set<String> hosts) {
+  final sanitizedHosts = hosts
+      // ignore: invalid_use_of_internal_member
+      .map((e) => sanitizeHost(e, datadog.internalLogger))
+      .whereType<String>();
+  final allowedWebViewHostsString = sanitizedHosts.map((e) => '"$e"').join(',');
+
+  String bridgeSource = '''
+/* DatadogEventBridge */
+window.DatadogEventBridge = {
+  cache: [],
+  send(msg) {
+    if (window.$bridgeName) {
+      if (this.cache.length > 0) {
+        this.cache.forEach((e) => window.$bridgeName.callHandler('$handlerName', msg))
+        this.cache = []
+      }
+      window.$bridgeName.callHandler('$handlerName', msg)
+    } else {
+      this.cache.push(msg)
+    }
+  },
+  getAllowedWebViewHosts() {
+      return '[$allowedWebViewHostsString]'
+  },
+  getCapabilities() {
+      return '[]'
+  },
+  getPrivacyLevel() {
+      return 'mask'
```

Comment:
should this be a configuration option?

Reply:
No, this is used for SR only, which we don't support. This is the default.